### ISSUE TITLE
Option to exclude certain metadata fields

### ISSUE THREAD

**Issue by zmwangx** [zmwangx/storyboard#8]
This isn't a priority, but I'll support this if I have time.

An easy (but probably not optimal) way to support this is through an argument `excluded_fields` in the relevant API(s) which takes a set/list of field names. If there are too many fields in total, it is also possible to have another option `fields` to include rather than exclude fields.

I'd rather exclude fields at the time of string generation rather at the time of crunching metadata, since crunching metadata is pretty fast anyway (except SHA-1, which always requires a separate option). Therefore, the field selection options should be implemented in the output interface rather than init interface.

This is blocked by #6.

**Comment by zmwangx** [comment]
#6 is fixed, and the APIs are very extensible now. So this is already possible, and probably not hard.

**Comment by zmwangx** [comment]
`--include` and `--exclude`. I'll implement them soon before it's too late.

**Comment by zmwangx** [comment]
The plan:
- `metadata.Video.format_metadata`: the `include_sha1sum` option will be kept as a shortcut, but it will two new options, `exclude` and `include`, which are two lists of field names. `include_sha1sum` meres adds to the list of includes. `include` always overwrites `exclude`. There will be a special name `all` which just means all fields, so one can exclude all and include incrementally.
  
  All fields are included in the beginning. `None` is skipped. The SHA-1 field is treated specially in that by default it is only picked up if `self.sha1sum` is not `None` (i.e., been computed before); however, if it appears in the `include` list, then it is computed explicitly.
- `metadata` CLI: similiarly, `-s,--include-sha1sum` will be kept as a shortcut, but there will be two new options `--include` and `--exclude` (no short option since `-i` and `-e` are frequently used for other things, e.g., `--input`, `--enable`/`--escape`/`--regexp`, so using them will be pretty confusing). `--include` and `--exclude` should be able to accept comma-separated lists or be specified multiple times. They will also earn their places in the config file, not needless to say, only once.
- `storyboard` CLI: same thing.

**Comment by zmwangx** [comment]
I think I'm too tired to implement this. I certainly won't need this myself. Marking as wontfix for now, and will remain wontfix unless there's increased interest in this package in the future (highly unlikely). Same goes for exposing all storyboard parameters in the CLI of `storyboard.storyboard`.
